source/qml_v2/AdaptiveCardQmlEngine/qml/AdaptiveCard.qml

```diff
@@ -96,6 +96,7 @@ Rectangle {
     onCardThemeChanged: {
         adaptiveCardController.setCardTheme(cardTheme)
         adaptiveCardController.setCardJSON(cardJSON)
+
```

Comment:
Why this change ?

Reply:
There is no changes only extra space was there. I'll remove it.